components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -284,6 +284,10 @@ paths:
           required: true
           schema:
             $ref: '#/definitions/API'
+        - in: query
+          name: oas_version
```

Comment:
Shall we use 'oasVersion' or 'openAPIVersion'? We usually don't use _s in parameters.

Reply:
fixed